### PR TITLE
use building block as a static page

### PR DESCRIPTION
**Pull request by kain88-de**
This should be enough to have a static page. I cannot test right now because my bundler installation is broken.

### CODE REVIEW

**Comment by kain88-de** [comment]
ok fixed jekyll. It renders as a static page

File: pages/mdanalysis-as-a-building-block.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+layout: page
+title: MDAnalysis as a building block
```
Comment (orbeckst) [inline pages/mdanalysis-as-a-building-block.md]: This title shows up in the menu bar. It is quite long. I'd like something shorter, along the lines of
- Who uses MDAnalysis?
- Made with MDA
- Uses

I don't particularly like having "MDAnalysis" in the title.

Do you have some better ideas?

Comment (kain88-de) [reply pages/mdanalysis-as-a-building-block.md]: maybe "Used By"?

**Comment by orbeckst** [comment]
I like it!

**Comment by kain88-de** [comment]
is it just me or does this change not render on the webpage

**Comment by orbeckst** [comment]
Nope, I don't see it either. Locally it rendered, I had checked. 

Maybe GitHub has issues with their Jekyll. 

Debugging gh pages is difficult.

**Comment by kain88-de** [comment]
it rendered for me too. Maybe it will pop up with the next blog update.
There are some coming up for gsoc anyway

**Comment by orbeckst** [comment]
@kain88-de the "Use by" static page is now showing up.

**Pull request merged**